app/connectors/TrustDataConnector.scala

```diff
@@ -40,15 +41,15 @@ class TrustDataConnector @Inject()(http: HttpClient, config: AppConfig) extends
       CORRELATION_ID -> correlationId
     )
 
-  def getTrustJson(identifier: Identifier)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[TrustDataResponse] = {
+  def getTrustJson(identifier: Identifier): Future[TrustDataResponse] = {
     lazy val url: String = s"${config.trustDataUrl}/trusts/obliged-entities/$identifier/${identifier.value}"
 
     val correlationId = UUID.randomUUID().toString
 
-    val headersWithoutOldAuth = hc.copy(authorization = None)
-    val hcExtra: HeaderCarrier = headersWithoutOldAuth.withExtraHeaders(trustDataHeaders(correlationId): _*)
+    implicit val hc: HeaderCarrier = HeaderCarrier(authorization = None, extraHeaders = trustDataHeaders(correlationId))
     logger.info(s"[Session ID: ${Session.id(hc)}] getTrustJson correlationId: $correlationId from call to url: $url")
```

Comment:
Is this log now always going to return "No Session ID available" ?

Reply:
Hmmm. It looks like it probably will, but I basically just copied it from Trusts. For my logging, it's the CorrelationId I really care about, so I don't mind